File: Codeforces/Codeforces-1183A.cpp
#include<bits/stdc++.h>

using namespace std;


#define MAX 100010
#define LL long long int

bool check(int n)
{
    int res = 0;
    while(n)
    {
        res = res + n%10;
        n /= 10;
    }
    return res%4 == 0? 1 : 0;
}

int main()
{
    int n;
    cin >> n;
    while(!check(n))
    {
        n++;
    }
    cout << n << endl;
    return 0;
}

File: Python/Codeforces-1183A.py
def check(n):
    res = 0
    while n != 0:
        res = res+n%10;
        n = n//10;
    if res%4 == 0:
        return 1
    else:
        return 0


n = int(input())
while not check(n):
    n = n+1
print(n)